api/express.js

```diff
@@ -44,7 +48,18 @@ app.use(cors());
 app.use(limiter);
 app.use('/api/docs', swaggerUi.serve, swaggerUi.setup(swaggerJSDoc(OPENAPI_OPTIONS)));
 
-// Routing
-app.use('/api', apiRouter);
+// Routing Open Routes
+app.use('/api', openRouter.chefsRouter);
+app.use('/api', openRouter.healthRouter);
+
+// Routing Protected Routes
+// Allow for removed protection when API testing
+
+// TODO: Remove test route after demo
+if (`${process.env.TESTING}`.toLowerCase() == "true") {
```

Comment:
You could also do this as 

``` JavaScript
app.use('/api', process.env.TESTING ? () => {} : protect, protectedRouter.keycloakTest);
```

Reply:
Did something similar. It didn't like the empty function, but I could mock a middleware function and that seems alright.